event-horizon-cli/build.gradle.kts

```diff
@@ -8,6 +8,14 @@ application {
   mainClass.set("com.automattic.eventhorizon.cli.MainKt")
 }
 
+graalvmNative {
+  binaries {
+    named("main"){
```

Comment:
Could we add `./gradlew spotlessCheck` to the CI pipeline? Not necessarily in this PR but this would automate code formatting.